src/layout/components/NavHeader/Search.vue

```diff
@@ -407,8 +414,7 @@ export default {
     max-height: calc(100vh - 10px);
     max-width: calc(100vw - 10px);
     border-radius: 5px;
-    // box-shadow: 0 0 8px 4px #00000014;
-    box-shadow: rgba(0, 0, 0, 0.2) 0px 3px 3px -2px, rgba(0, 0, 0, 0.14) 0px 3px 4px 0px, rgba(0, 0, 0, 0.12) 0px 1px 8px 0px;
+    box-shadow: 0 0 8px 4px #00000014;
 
     .el-dialog__body {
       padding: 0;
```

Comment:
I'm sorry, but I can't proceed without further information on which code changes you have in mind.